### PR TITLE
Add note on OpenSSL version in TLS certs doc

### PR DESCRIPTION
**Pull request by 5abeel**
Adding some clarity to make sure users are aware of the OpenSSL versions and are not mismatching versions between client and server systems since there are failures noted when trying to use OpenSSL 1.1.1x client with a OpenSSL 3.x server

### CODE REVIEW

File: docs/guides/security/using-tls-certificates.md [hunk 1]
```diff
@@ -24,6 +24,14 @@ COMMON_NAME=<IP> ./generate-certs.sh
 
 The system relies on mTLS (mutual TLS) for authentication.
 
+### OpenSSL version
+
+The `/usr/share/stratum/generate-certs.sh` script uses the installed OpenSSL version to generate the certificates. 
+
+OpenSSL 1.1.1x has reached EOL and usage should be discontinued. See the [OpenSSL security guide](openssl-guide.md) for details.
+
+Also, note that if running gRPC clients on remote system, both systems are running OpenSSL 3.x. Running an OpenSSL 1.1.1x client with a OpenSSL 3.x server has known to fail TLS handshakes to establish communication.
```
Comment (nupuruttarwar) [inline docs/guides/security/using-tls-certificates.md:33]: both systems should be running OpenSSL 3.x

Comment (nupuruttarwar) [inline docs/guides/security/using-tls-certificates.md:33]: Running an OpenSSL 1.1.1x client with a OpenSSL 3.x server has been known to fail TLS handshakes with `WRONG_VERSION_NUMBER` error when trying to establish communication?

Comment (5abeel) [reply docs/guides/security/using-tls-certificates.md]: Fixed

Comment (5abeel) [reply docs/guides/security/using-tls-certificates.md]: Fixed

**Review by nupuruttarwar — APPROVED** [review]
LGTM

**Review by ffoulkes — APPROVED** [review]
LGTM

**Pull request merged**